backend/src/akvo/lumen/lib/tier_impl.clj

```diff
@@ -3,7 +3,9 @@
    [ring.util.response :refer [response]]))
 
 (defn all [tenant-conn]
-  (response {"tiers" {"standard" {"numberOfExternalDatasets" 5
+  (response {"tiers" {"unlimited" {"numberOfExternalDatasets" "*"
```

Comment:
What we render is different from what we the data structure contains. I was just thinking that it's a bit strange to have key/value pairs where the the value is sometimes a number and sometimes a string. `null` to me would mean "no value" and would therefor be unlimited.

We can still render *, unlimited, ∞ or whatever